### ISSUE TITLE
show the first line of the file contents in the file tab for unsaved file

### ISSUE THREAD

**Issue by modernNeo** [atom/atom#18978]
## Summary
I am trying to transition from Sublime which has a feature I like where if a file is not yet saved, instead of showing the string "Untitled" as Atom does in the file tab or something similar, it instead displays as much of the first line as is possible on the file tab. I tried to look through all of atom's customisation for this feature and couldn't find it. Did I miss it or is it something that Atom does not have?
### Sublime Text
![Screenshot from 2019-03-12 20-37-29](https://user-images.githubusercontent.com/14242088/54252183-18857d80-4507-11e9-816c-2cb83cef2c9a.png)
### Atom
![Screenshot from 2019-03-12 20-37-58](https://user-images.githubusercontent.com/14242088/54252348-ab261c80-4507-11e9-9259-279b684433cb.png)

## Motivation

Why are we doing this? What use cases does it support? What is the expected outcome?

It provides a nice way to distinguish each unsaved files from each other rather than having to go through each unsaved file specifically to see if it has the contents we want.

## Describe alternatives you've considered

A clear and concise description of the alternative solutions you've considered. Be sure to explain why Atom's existing customizability isn't suitable for this feature.

I am unsure if any alternatives exist, if they do, please feel free to close this Issue and point me in the right direction.

**Comment by rsese** [comment]
Thanks for the suggestion @modernNeo!  While this does seem like it could be useful for some people, as mentioned in the issue template the team is unlikely to work on feature requests right now (in favor of things like bug fixes).  Additionally:

> I am unsure if any alternatives exist, if they do, please feel free to close this Issue and point me in the right direction.

I searched a bit and found this package that seems to do what you want:

https://atom.io/packages/tab-title

I installed it and tried it quickly and it looks like it works.

So since this isn't something the team would work on and since the functionality can be implemented outside of core Atom (and is available now with https://atom.io/packages/tab-title) , we'll go ahead and close this.
